charts/ks-devops/charts/jenkins/values.yaml

```diff
@@ -453,4 +453,4 @@ securityRealm:
 
 ksAuth:
   enabled: false
-#  server:
+  server: ""
```

Comment:
How users can know what the server is?

Reply:
Users could check server from [here](https://github.com/kubesphere-sigs/ks-devops/blob/1102edae144c1dbe4c2005059be9e3b7daaa2bb2/charts/ks-devops/templates/service-apiserver.yaml):

Reply:
> Users could check server from [here](https://github.com/kubesphere-sigs/ks-devops/blob/1102edae144c1dbe4c2005059be9e3b7daaa2bb2/charts/ks-devops/templates/service-apiserver.yaml).

Why do they should know it? Is there any document for it?

The most important thing is that it should work well without any extra parameters. We must make it be simple as possible as we can.

Reply:
We can pass the ksAuth.server outside of charts as below:

```bash
helm install ks-devops -n kubesphere-devops-system xyz
    --set jenkins.ksAuth.enabled=true \
    --set jenkins.ksAuth.server=http://ks-devops-apiserver.kubesphere-devops-system:9090
```

Reply:
Let's assume I don't have too much knowledge about helm charts and k8s. Then I don't know how to provide `http://ks-devops-apiserver.kubesphere-devops-system:9090`.

Reply:
Now, `ksAuth.eanble` config is set with `false` in `Makefile`, and there is no need to concern `ksAuth.server` config when someone runs the command `make install-chart`.

Reply:
Please answer my question:

Let's assume I don't have too much knowledge about helm charts and k8s. Then I don't know how to provide http://ks-devops-apiserver.kubesphere-devops-system:9090.

Reply:
I'will add another documentation to describe how to develop  in KubeSphere in the future.

Reply:
> I'will add another documentation to describe how to develop in KubeSphere in the future.

I cannot accept it. Users don't need to know about it. Because it's possible to provide a default value. Please see the format:

`http://{ks-devops-chartname}-apiserver.{ks-devops-namespace}:9090`